src/__tests__/element-queries.js

```diff
@@ -533,14 +533,14 @@ test('queryAllByRole returns semantic html elements', () => {
   expect(queryAllByRole(/listitem/i)).toHaveLength(3)
   // TODO: with https://github.com/A11yance/aria-query/pull/42
   // the actual value will match `toHaveLength(2)`
-  expect(queryAllByRole(/textbox/i)).toHaveLength(1)
+  expect(queryAllByRole(/textbox/i)).toHaveLength(2)
   expect(queryAllByRole(/checkbox/i)).toHaveLength(1)
   expect(queryAllByRole(/radio/i)).toHaveLength(1)
   expect(queryAllByRole('row')).toHaveLength(3)
   expect(queryAllByRole(/rowgroup/i)).toHaveLength(2)
   // TODO: with https://github.com/A11yance/aria-query/pull/42
   // the actual value will match `toHaveLength(3)`
```

Comment:
I guess these comments (this one and the one in lines 534-535) can be removed now 🎉

Reply:
https://github.com/testing-library/dom-testing-library/pull/665